components/raftstore/src/store/metrics.rs

```diff
@@ -489,6 +489,12 @@ lazy_static! {
     pub static ref STORE_SNAPSHOT_VALIDATION_FAILURE_COUNTER: SnapValidVec =
         auto_flush_from!(STORE_SNAPSHOT_VALIDATION_FAILURE_COUNTER_VEC, SnapValidVec);
 
+    pub static ref STORE_SNAPSHOT_RETIRES_COUNTER: IntCounter =
```

Comment:
Why not reuse existing metrics?

Reply:
cannot find a name suitable existing metrics